src/plugins/intel_cpu/tests/functional/subgraph_tests/src/instances/x64/conv_maxpool_activ.cpp

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "subgraph_tests/src/classes/conv_maxpool_activ.hpp"
+#include "test_utils/fusing_test_utils.hpp"
+#include "test_utils/filter_cpu_info.hpp"
+#include "ov_models/builders.hpp"
+
+using namespace ngraph;
+using namespace InferenceEngine;
```

Comment:
Please, remove all `InferenceEngine` usage

Reply:
InferenceEngine and ngraph namespaces are being using in 100+ files in CPU plugin. May we refactor subgraph tests in the dedicated PR? I assume @olpipi is already working on tests refactoring.

Reply:
No

Reply:
@vurusovs please give me an example of such PR

Reply:
@vurusovs @olpipi I'd suggest to enable subgraph tests for arm (merge this PR) firstly and then open a new one to refactor tests.
Otherwise this PR will never be merged taking into account it affects all CPU plugin subgraph tests.

@dmitry-gorokhov is it strictly required to refactor subgraph tests (remove `InferenceEngine` namespace usage) to merge this PR?